pallets/ddc-staking/src/lib.rs

```diff
@@ -861,6 +866,46 @@ pub mod pallet {
 			<Pricing<T>>::set(Some(price_per_byte));
 			Ok(())
 		}
+
+		/// Add a new account to the list of cluster managers.
+		///
+		/// RuntimeOrigin must be Root to call this function.
+		#[pallet::weight(T::WeightInfo::allow_cluster_manager())]
+		pub fn allow_cluster_manager(
+			origin: OriginFor<T>,
+			grantee: <T::Lookup as StaticLookup>::Source,
+		) -> DispatchResult {
+			ensure_root(origin)?;
+
+			let grantee = T::Lookup::lookup(grantee)?;
+			ClusterManagers::<T>::mutate(|grantees| {
+				if !grantees.contains(&grantee) {
+					grantees.push(grantee);
+				}
+			});
+
```

Comment:
Usually, it's a good practice to `deposit_event` before the return. At least to make it visible in the explorer.

Reply:
Events not required by the user.